doc/TSCC.rst

```diff
@@ -242,10 +255,36 @@ Get Slack notifications when your jobs finish
 
     slack "your job terminated with exit status $?"
 
+Installing software
+-------------------
+The best practice is for each user of TSCC to use Miniconda to install their own software. Run these commands to download, install, and configure Miniconda properly on TSCC:
+
+.. code-block:: bash
+
+  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+  bash Miniconda3-latest-Linux-x86_64.sh -b -u
+  source ~/miniconda3/bin/activate
+  conda init bash
+  conda config --remove channels defaults
+  conda config --add channels nodefaults
+  conda config --add channels bioconda
+  conda config --add channels conda-forge
+  conda config --set channel_priority strict
+
+If you are feeling lazy, you can also use :code:`module` system to load preconfigured software tools.
+Refer to `the TSCC documentation <https://www.sdsc.edu/support/user_guides/tscc.html#env_modules>`_ for more information.
+Please note that software available through the module system is usually out of date and cannot be easily updated.
+It's also unlikely that your collaborators/reviewers will be able to figure out which versions of the software you used.
+(Unlike with conda, there isn't a way to share your module environments with non-TSCC users.)
+For these reasons, we do not recommend using the :code:`module` system.
```

Comment:
```suggestion
.. warning::
  Software available through the module system is usually out of date and cannot be easily updated.
  It's also unlikely that collaborators/reviewers will be able to run your code once you're ready to share it with them, since,
  unlike with conda, there isn't a way to share your module environments with non-TSCC users.
  For these reasons, we do not recommend using the :code:`module` system.
```